content/blog/rabbitmq-kafka-revisited.md

```diff
@@ -0,0 +1,69 @@
+
+## Requirements and Use Cases
+
+When the initial blog post was written, there was a pretty clear-cut difference in design between RabbitMQ and Kafka, and as such, a difference in use cases. RabbitMQ’s message broker design excelled in use cases that had specific routing needs and per-message guarantees, whereas Kafka’s append-only log allowed developers access to the stream history and more direct stream processing. While the Venn diagram of use cases these two technologies could fulfill was very tight, there were scenarios in which one was a demonstrably better choice than the other.
+
+Work is currently underway that will alter  that balance, however. While RabbitMQ will continue to offer its traditional queue model, it will also  introduce a new data structure modeling an append-only log, with non-destructive consuming semantics. This new data structure will work much like Kafka’s persistent log, and will be an exciting addition for RabbitMQ users looking to expand their streaming use case. And while this feature will be compatible with the AMQP protocol, it will also introduce a binary-based stream protocol. 
+
+## Developer Experience
+
+The developer experience of the two services has largely remained the same, with the list of clients and libraries continuing to grow thanks to the work of their respective communities. Both [RabbitMQ’s](https://www.rabbitmq.com/devtools.html) and [Kafka’s](https://cwiki.apache.org/confluence/display/KAFKA/Clients) client library lists have seen steady growth. As more languages and frameworks have grown in popularity, finding a well-supported and complete library for either service has become easier. 
+
+One thing to note is the growth of [Kafka Streams](https://kafka.apache.org/documentation/streams/), a client library implementation that makes it easier for developers to process streaming data. It’s used for the common use case of reading data from Kafka, processing it, and writing it to another Kafka queue. Additionally, [ksqlDB](https://ksqldb.io) provides a great way for developers to build streaming applications, and well worth checking out. 
+
+A similar thing can be accomplished with RabbitMQ with the help of some other pieces, such as [Spring Cloud Data Flow](https://dataflow.spring.io/docs/stream-developer-guides/streams/standalone-stream-rabbitmq/). Furthermore, note the upcoming streaming changes coming for RabbitMQ mentioned in the previous section, keeping in mind that this can open new ways of interacting with RabbitMQ for the developer.
+
+## Security and Operations
+
+As noted in the initial post, RabbitMQ ships with a useful administration interface to manage users and queues, while Kafka relies on TLS and JAAS. Whether you choose RabbitMQ or Kafka will of course depend on your specific requirements and your use case, but most security scenarios can have a proper conclusion with either technology. 
```

Comment:
Can we link TLS and JAAS off somewhere and/or spell them out? TLS is hopefully familiar to anyone reading this, but JAAS might not be?